source/common/tcp_proxy/tcp_proxy.cc

```diff
@@ -292,6 +291,11 @@ void Filter::readDisableDownstream(bool disable) {
   }
 }
 
+StreamInfo::StreamInfo& Filter::getStreamInfo() {
+  ASSERT(read_callbacks_ != nullptr);
```

Comment:
nit: del as this will crash in an obvious way on the next line.

Reply:
Thanks.. Will remove